documentation/develop/getting-started-with-web-ext.md

```diff
@@ -4,16 +4,204 @@ title: Getting started with web-ext
 permalink: /documentation/develop/getting-started-with-web-ext/
 published: false
 topic: Develop
+tags: ['test', 'community', 'hello']
+contributors: [caitmuenster, muffinresearch, bqbn]
+author: caitmuenster
+date: 2019-06-24 09:00:00
 ---
 
-<section class="module panel">
+<!-- Page Hero Banner -->
+
+<section class="page-hero">
+<div class="module">
 <article class="module-content grid-x grid-padding-x">
-<div class="cell small-12" markdown="1">
+<div class="cell small-12">
+<div class="page-hero-description" markdown="1">
+<p class="section-title"><small>{{ page.topic }}</small></p>
 
 # Getting started with web-ext
 
 web-ext is a command line tool designed to speed up various parts of the extension development process, making development faster and easier. This article explains how to install and use web-ext.
 
 </div>
+<div class="page-hero-cta">
+
+<a href="/documentation/develop/getting-started-with-web-ext/" class="button">Do first thing</a>
+<a href="/documentation/develop/build-an-extension-in-5-minutes" class="button secondary">Do second thing</a>
+
+</div>
+</div>
+</article>
+</div>
+</section>
+
+<!-- END: Page Hero Banner -->
+
+<!-- Single Column Body Module -->
+
+<section id="installation-section" class="module">
+<aside class="module-aside table-of-contents" markdown="1">
+
+###### Contents
+
+1. [Installation](#installation-section 'Installation')
+2. [Using web-ext](#using-web-ext-section 'Using web-ext')
+3. [See also](#see-also-section 'See also')
+
+</aside>
+<article class="module-content grid-x grid-padding-x">
+<div class="cell small-12" markdown="1">
+
+## Installation
+
+`web-ext` is a node-based application that you install with the <a href="https://www.firefox.com" target="_blank" rel="noreferrer noopener" title="nodejs/npm">nodejs/npm</a> tool. Install web-ext using the following command:
```

Comment:
Presumably these can be updated to regular markdown links and have the plugin handle the `rel="noopener noreferrer"` and target attrs?